_posts/pl/2021-09-16-bucket-pattern.md

```diff
@@ -0,0 +1,88 @@
+---
+layout:    post
+title:     "Czy wiesz, czym jest Bucket Pattern w MongoDB i czemu przydał się np. w implementacji IoT Boscha lub w aplikacjach bankowych?"
+date:      2021-11-03 6:00:00 +0100
+published: true
+didyouknow: true
+lang: pl
+author:    bmitan
+image:     /assets/img/posts/2021-09-16-bucket-pattern/bucket.jpeg
+tags:
+- mongoDB
+---
+
+## Jaki problem próbujemy rozwiązać?
+
+Zacznijmy od prostego przykładu - **pomiarów czujnika wykonywanych bardzo często**, np. co sekundę, każdego dnia. Można by zapisywać je np. następująco:
```

Comment:
skoro mamy już sformułowanie "można by zapisywać je"..to z np. bym już zrezygnowała, czyli:
Można by je zapisywać następująco:

Reply:
done